src/mutation/RelayMutationQuery.js

```diff
@@ -23,6 +23,8 @@ const RelayQuery = require('RelayQuery');
 import type RelayQueryTracker from 'RelayQueryTracker';
 const RelayRecord = require('RelayRecord');
 import type {Variables} from 'RelayTypes';
+const GraphQLMutatorConstants = require('GraphQLMutatorConstants');
+const REFETCH = GraphQLMutatorConstants.REFETCH;
```

Comment:
Could do:

    const {REFETCH} = GraphQLMutatorConstants;

of even collapse the two lines:

    const {REFETCH} = require('GraphQLMutatorConstants');